frontend/src/scenes/error-tracking/errorTrackingDataNodeLogic.tsx

```diff
@@ -22,48 +22,48 @@ export const errorTrackingDataNodeLogic = kea<errorTrackingDataNodeLogicType>([
     })),
 
     actions({
-        mergeGroups: (indexes: number[]) => ({ indexes }),
-        assignGroup: (recordIndex: number, assigneeId: number | null) => ({
+        mergeIssues: (indexes: number[]) => ({ indexes }),
+        assignIssue: (recordIndex: number, assigneeId: number | null) => ({
             recordIndex,
             assigneeId,
         }),
     }),
 
     listeners(({ values, actions }) => ({
-        mergeGroups: async ({ indexes }) => {
-            const results = values.response?.results as ErrorTrackingGroup[]
+        mergeIssues: async ({ indexes }) => {
+            const results = values.response?.results as ErrorTrackingIssue[]
 
-            const groups = results.filter((_, id) => indexes.includes(id))
-            const primaryGroup = groups.shift()
+            const issues = results.filter((_, id) => indexes.includes(id))
```

Comment:
the use of `id` throughout this function through me off - in most cases, you really mean an `index` in the array underlying the table this is the logic for, right?

Reply:
Updated given we now _actually_ use ids